mifosng-provider/src/main/java/org/mifosplatform/portfolio/group/service/GroupingTypesWritePlatformServiceJpaRepositoryImpl.java

```diff
@@ -792,6 +803,10 @@ public CommandProcessingResult disassociateGroupsToCenter(final Long centerId, f
 
         final Map<String, Object> actualChanges = new HashMap<>();
 
+		for (final Group group : groupMembers) {
+			group.resetHierarchy();
```

Comment:
move this to public List<String> disassociateGroups(Set<Group> groupMembersSet) { in Group.java